runtime/auth/aws-signing-tests/jvm/src/aws/smithy/kotlin/runtime/auth/awssigning/tests/SigningSuiteTestBaseJVM.kt

```diff
@@ -89,7 +103,17 @@ actual abstract class SigningSuiteTestBase : HasSigner {
             .map { it.parent }
     }
 
-    protected open val disabledTests = setOf<String>()
+    protected open val disabledTests = setOf<String>(
+        // ktor-http-cio parser doesn't support parsing multiline headers since they are deprecated in RFC7230
+        "get-header-value-multiline",
+        // ktor fails to parse with space in it (expects it to be a valid request already encoded)
+        "get-space-normalized",
```

Comment:
We should probably address these as they are mostly limitations of the parser in use. In particular the space ones since IIRC rust had some issues with this. I don't think we need to block this PR over it but we should make a task for it.

Reply:
Yep yep, created #653 and will note it in a TODO around these tests.